deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -172,6 +173,13 @@ def run_logic(self, context: Context) -> CheckResult:
                 train_prediction = train_prediction[:, [1]]
                 test_prediction = test_prediction[:, [1]]
         else:
+            if context.task_type == TaskType.MULTICLASS:
+                get_logger().warning(
+                    'deepchecks infers label to be multiclass and auto plot class prediction drift. '
+                    'It is recommended to plot prediction probabilities instead, in order to '
+                    'produce meaningful results. To do so, add "drift_mode=\'proba\'" '
+                    'to TrainTestPredictionDrift check'
```

Comment:
I know that in #1820 there was a request for a validation of some sort, but this seems unnecessary, and I tried to discuss this with others and we can't currently find how this could cause confusion that the other warning you added doesn't cover.

Reply:
i agree, will change it